#Exception 是所有异常的父类
except Exception as e:
    print("出错了：", e)
